package/custom_project.conf

```diff
@@ -116,10 +127,9 @@
 </match>
 
 
-
-<filter tmp1.tmp2-project-custom.**>
+<filter tmp1.tmp2-project-custom.**> 
   @type kubernetes_metadata 
-  tag_to_kubernetes_name_regexp tmp1\.tmp2-project-custom\.var\.lib\.rancher\.log-volumes\.((?<pod_id>[^_]+)_(?<volume_name>[^_]+))\.(?<format>[a-z0-9]+)\.(?<cluster_id>[^_]+)_(?<cluster_name>[^_]+)_(?<namespace>[^_]+)_(?<project_id>[^_]+)_(?<project_name>[^_]+)_(?<workload_name>[^_]+)_(?<pod_name>[a-z0-9]([-a-z0-9]*[a-z0-9])*)_(?<container_name>[^_^.]+)\.(?<filename>[a-z0-9A-Z]+\.[a-zA-Z0-9]+)
+  tag_to_kubernetes_name_regexp tmp1\.tmp2-project-custom\.(?<pod_name>[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*)_(?<namespace>[^_]+)_(?<docker_id>.+)_(?<container_name>.+)$
```

Comment:
are we missing some info like `volume_name`, `pod_id `, `cluster_id ` and `project_id`? or we just move them somewhere else.

Reply:
ok, got it.